scripts/scr_battle_roster/scr_battle_roster.gml

```diff
@@ -121,87 +131,87 @@ function scr_battle_roster(_battle_loci, _loci_specific, _is_planet) {
 
-	                    if (obj_ini.role[cooh,va]="Codiciery"){col=obj_controller.bat_librarian_column;obj_ncombat.librarians+=1;moov=1;}
-	                    if (obj_ini.role[cooh,va]="Epistolary"){col=obj_controller.bat_librarian_column;obj_ncombat.librarians+=1;moov=1;}
-	                    if (obj_ini.role[cooh,va]="Lexicanum"){col=obj_controller.bat_librarian_column;obj_ncombat.librarians+=1;moov=1;}
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,16]){col=obj_controller.bat_techmarine_column;obj_ncombat.techmarines+=1;moov=2;}
+	                    if (deploying_unit.role[cooh,va]="Codiciery"){col=obj_controller.bat_librarian_column;obj_ncombat.librarians+=1;moov=1;}
+	                    if (deploying_unit.role[cooh,va]="Epistolary"){col=obj_controller.bat_librarian_column;obj_ncombat.librarians+=1;moov=1;}
+	                    if (deploying_unit.role[cooh,va]="Lexicanum"){col=obj_controller.bat_librarian_column;obj_ncombat.librarians+=1;moov=1;}
+	                    if (deploying_unit.role[cooh,va]=deploying_unit.role[100,16]){col=obj_controller.bat_techmarine_column;obj_ncombat.techmarines+=1;moov=2;}
 
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,2]){col=obj_controller.bat_honor_column;obj_ncombat.honors+=1;}
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,6]){col=obj_controller.bat_dreadnought_column;obj_ncombat.dreadnoughts+=1;}
-	                    if (obj_ini.role[cooh,va]="Venerable "+string(obj_ini.role[100,6])){col=obj_controller.bat_dreadnought_column;obj_ncombat.dreadnoughts+=1;}
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,4]){col=obj_controller.bat_terminator_column;obj_ncombat.terminators+=1;}
+	                    if (deploying_unit.role[cooh,va]=deploying_unit.role[100,2]){col=obj_controller.bat_honor_column;obj_ncombat.honors+=1;}
+	                    if (deploying_unit.role[cooh,va]=deploying_unit.role[100,6]){col=obj_controller.bat_dreadnought_column;obj_ncombat.dreadnoughts+=1;}
+	                    if (deploying_unit.role[cooh,va]="Venerable "+string(deploying_unit.role[100,6])){col=obj_controller.bat_dreadnought_column;obj_ncombat.dreadnoughts+=1;}
+	                    if (deploying_unit.role[cooh,va]=deploying_unit.role[100,4]){col=obj_controller.bat_terminator_column;obj_ncombat.terminators+=1;}
 
 	                    if (moov>0){
 	                        if ((moov=1) and (obj_controller.command_set[8]=1)) or ((moov=2) and (obj_controller.command_set[9]=1)){
 	                            if (company>=2) then col=obj_controller.bat_tactical_column;
 	                            if (company=10) then col=obj_controller.bat_scout_column;
-	                            if (obj_ini.mobi[cooh,va]="Jump Pack"){ col=obj_controller.bat_assault_column;}
+	                            if (deploying_unit.mobi[cooh,va]="Jump Pack"){ col=obj_controller.bat_assault_column;}
```

Comment:
mobi?

Reply:
it's an obscure attribute for checking if marines have bikes or jump packs but it's hard coded in the ini file so just has to be endured until fully refactored... I think